dash/examples/bench.13.allreduce/main.cpp

```diff
@@ -0,0 +1,223 @@
+/**
+ * Measures the performance of different
+ * for_each implementations on dash containers
+ */
+
+#include <libdash.h>
+#include <iostream>
+#include <iomanip>
+#include <string>
+
+using std::cout;
+using std::endl;
+using std::setw;
+using std::setprecision;
+
+typedef dash::util::Timer<
+          dash::util::TimeMeasure::Clock
+        > Timer;
+
+typedef typename dash::util::BenchmarkParams::config_params_type
+  bench_cfg_params;
+
+typedef struct benchmark_params_t {
+  int    reps;
+  int    rounds;
+} benchmark_params;
+
+typedef struct measurement_t {
+  std::string testcase;
+  double      time_total_s;
+} measurement;
+
+void print_measurement_header();
+void print_measurement_record(
+  const bench_cfg_params & cfg_params,
+  measurement              measurement,
+  const benchmark_params & params);
+
+benchmark_params parse_args(int argc, char * argv[]);
+
+void print_params(
+  const dash::util::BenchmarkParams & bench_cfg,
+  const benchmark_params            & params);
+
+
+measurement evaluate(
+              int reps,
+              std::string testcase,
+              benchmark_params params);
+
+int main(int argc, char** argv)
```

Comment:
Should we even keep this? No one will probably run it again ;-)

Reply:
Yes, I would keep it, it's interesting to see and could be used for evaluation later.